package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "@canonical/juju-react-components",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "React components for use with Juju",
   "main": "index.js",
+  "source": true,
   "scripts": {
     "lint": "eslint .",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test": "jest"
+    "test": "jest",
+    "update-snapshot": "jest --updateSnapshot"
```

Comment:
I'm not sure we want to do this, typically I update the snapshots on a case by case basis. Don't want to update the snapshot if there is indeed a failure in the code.

Reply:
NPM also allows arbitrary parameters so the test command could be used instead e.g. `npm test -- --updateSnapshot`. I've been using this a lot with `jest --watch` e.g. `npm test -- --watch`.

Reply:
@huwshimi cool, I didn't know that

Reply:
Cool, indeed! Ok removed that line.